frontend/src/metabase/actions/components/ActionForm/ActionForm.tsx

```diff
@@ -119,11 +116,7 @@ export const ActionForm = ({
   if (isSettings) {
     const fieldSettings = formSettings.fields || {};
     return (
-      <FormProvider
-        initialValues={initialValues}
-        validationSchema={formValidationSchema}
-        onSubmit={handleSubmit}
-      >
+      <FormProvider initialValues={initialValues} onSubmit={handleSubmit}>
```

Comment:
Turning off form validation inside the action editor as it doesn't make a lot of sense

Reply:
It will turn off the form validation in all cases, including when running the action via Run button or in a dashboard

Reply:
Actually not, but I can see why it is so confusing. The thing is that `ActionForm` can render two pretty different forms depending on the `isSettings` flag. The PR patches [`isSettings === true` scenario](https://github.com/metabase/metabase/blob/e09ecdb6e89fa9ea10b4d73b727c237a78182667/frontend/src/metabase/actions/components/ActionForm/ActionForm.tsx#L119) and runnable forms go through [another route just below](https://github.com/metabase/metabase/blob/e09ecdb6e89fa9ea10b4d73b727c237a78182667/frontend/src/metabase/actions/components/ActionForm/ActionForm.tsx#L178). We should have broken it down into two different components earlier and this is one of the main things I'd like to patch in a wider refactoring I've mentioned

<img width="663" alt="CleanShot 2023-03-03 at 17 24 55@2x" src="https://user-images.githubusercontent.com/17258145/222787012-d49d52bc-eb65-4bba-a39e-b2002b3bd152.png">